test/client/script.js

```diff
@@ -340,4 +340,76 @@ describe(`script cases`, () => {
             throw new Error(`Expected page type to be empty, got ${ getPageType() || 'undefined' } from ${ url }`);
         }
     });
+
+    it('should successfully get locale from script', () => {
+        const expectedLocale = 'en_US';
+
+        const url = insertMockSDKScript({
+            query: {
+                'locale': expectedLocale
+            }
+        });
+
+        const localeObject = getLocale();
+        const receivedLocal = `${ localeObject.lang }_${ localeObject.country }`;
+        if (expectedLocale !== receivedLocal) {
+            throw new Error(`Expected client id to be ${ expectedLocale }, got ${ receivedLocal } from ${ url }`);
+        }
+    });
+
+    it('should successfully get locale from browser settings', () => {
+        const expectedLocale = 'en_US';
+        const defineProp = Object.defineProperty;
+
+        defineProp(navigator, 'languages', {
```

Comment:
Thanks for adding tests @amyegan! 💯 

I've had to do the same with making globals writable in jest. One suggestion is to do this in the `beforeEach` like so:

```js
beforeEach(() => {
    Object.defineProperty(window.navigator, 'languages', { value: [], writable: true });
});

// and then in the test do the following
window.navigator.languages = ['en_US'];
```